src/CMakeLists.txt

```diff
@@ -35,8 +35,10 @@ endif()
 
 if(WIN32)
   # Enable DEP & ASLR
-  set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--nxcompat -Wl,--dynamicbase")
-  set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--nxcompat -Wl,--dynamicbase")
+  if (MINGW)
+    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,--nxcompat -Wl,--dynamicbase")
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--nxcompat -Wl,--dynamicbase")
```

Comment:
Meaning they did nothing? Do we already enable `/DYNAMICBASE` for msvc in another place then?

Reply:
Nope they are just the msvc default so setting them would be a noop.
Setting the gnu flags however creates a whole bunch of warnings.